### PR TITLE
Implemented basic scenarios

### PR DESCRIPTION
**Pull request by EnitanOO**
[AB#9258](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9258)
[AB#13215](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13215)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/OneListPriceForAdditionalService.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿Feature: One List Price For Additional Service Added
+    As a Buyer
+    I don't to want to have to select the list price of an Additional Service that has only one list price
+    So that I save time and effort when completing my order
+
+Background:
+	Given an incomplete order with catalogue items exists
+    When the Order Form for the existing order is presented
+    Then there is the Additional Service section
+	And the User is able to manage the Additional Services section
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/OneListPriceForAdditionalService.feature:10]: The spacing doesn't seem to be consistent here

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Very minor change

File: src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/OneListPriceForAdditionalService.feature (hunk 1 above)
Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Tests/Features/ManageOrderForm/OneListPriceForAdditionalService.feature:10]: sorting

**Review by PhysicsJon — APPROVED** [review]

**Comment by EnitanOO** [comment]
Thank you!

**Pull request merged**